# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/service-workers/service-worker/ServiceWorkerGlobalScope/registration-attribute.https.html.ini
# Commit: 244df402d6eb
# Full Hash: 244df402d6ebc15d6845c4144245511021408835
# Author: Andrew Sutherland <asutherland@asutherland.org>
# Date: 2024-10-24 09:44:34
# Regressor Bug: 1131324
# File Overlap Count: 1
# Description:
#   Bug 1131324 - Expose ServiceWorkerContainer on WorkerNavigator. r=dom-worker-reviewers,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D213725
# ==============================================================================

diff -r 57f70ba5f27e -r 244df402d6eb testing/web-platform/meta/service-workers/service-worker/ServiceWorkerGlobalScope/registration-attribute.https.html.ini
--- a/testing/web-platform/meta/service-workers/service-worker/ServiceWorkerGlobalScope/registration-attribute.https.html.ini	Thu Oct 24 03:02:39 2024 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,6 +0,0 @@
-[registration-attribute.https.html]
-  [Verify registration attributes on ServiceWorkerGlobalScope]
-    expected: FAIL
-
-  [Verify registration attributes on ServiceWorkerGlobalScope of the newer worker]
-    expected: FAIL